### ISSUE TITLE
Save "reply contact" information on permanent storage

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#137]
```
What steps will reproduce the problem?
1. chat with someone using gtalksms
2. use your phone to make a phone call (which causes gtalksms to log off, at 
least on verizon)
3. end phone call (which allows gtalksms to reconnect)
4. try to continue your previous sms conversation using the "reply" command
5. receive this message: Reply contact is not set.

What is the expected output?
the reply contact should not be reset, and the message should go to the 
previously used contact

What do you see instead?
Reply contact is not set.

Additional informations:
GTalkSMS version =2.1.2
Android OS version =2.2
Phone model =htc incredible

Paste GTalkSMS logs if they are relevant
(http://code.google.com/p/gtalksms/wiki/GenerateAppLog)


```

Original issue reported on code.google.com by `joshk...@gmail.com` on 5 Apr 2011 at 11:08

**Comment by GoogleCodeExporter** [comment]
```
was already on my todo list :)
```

Original comment by `fschm...@gmail.com` on 6 Apr 2011 at 6:43
- Changed title: **Save "reply contact" information on permanent storage**
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
http://code.google.com/r/fschmaus-gtalksms/source/detail?r=dd194defeb2a1174ca7ba
1f21a4fbe403fd9418f
```

Original comment by `fschm...@gmail.com` on 8 Apr 2011 at 11:23
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 11 Apr 2011 at 5:29
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.2
```

Original comment by `Florent....@gmail.com` on 24 Apr 2011 at 1:16
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
